hydra/oauth2.py

```diff
@@ -3,7 +3,6 @@
 
 from datetime import datetime, timedelta
 from urllib.parse import urljoin
-
```

Comment:
Don't delete blank lines if it's not relevant to the change. Keep the diff as clean as possible, and put those minor changes in another commit.